ci/playbooks/build_push_container_runner.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Run build_push_container role
+  hosts: "{{ cifmw_zuul_target_host | default('controller') }}"
+  tasks:
+    - name: Run build_push_container role
+      vars:
+        ci_framework_src_dir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"
+      ansible.builtin.command:
```

Comment:
with long commands like that, I'd use `ci_script`. That would also generate the reproducer script, allowing to track potential issues such as "oh, we delegate to a host, and the ansible_user_dir is different" and related potential headaches.

Reply:
(IF ci_script is available at that level...)

Reply:
Nope it's not, even when asking Zuul to pull the role to the executor it doesn't work. 

Maybe there is a way to make it work but now my reasons against have stacked up:

- It makes the stupid runner playbook more complex, I need to create directories etc. 
- It's not how the other playbooks that are used to call nested playbooks are written so goes against the standard. 
- ci_script is not available for the Zuul executor and we want to run as little as possible in that space for the same reasons you mention, reproducing and tracking down issues with node holds etc.

Reply:
Makes sense - I wasn't sure about the ci_script availability ;).